liblustreapi-types/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "liblustreapi-sys"
+name = "liblustreapi-types"
```

Comment:
Can you keep the name of the crate as `liblustreapi-sys`? It's a convention used for bindgen wrapper crates.

Reply:
nm